sale_line_returned_qty/__manifest__.py

```diff
@@ -1,13 +1,14 @@
 # Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# Copyright 2024 Alitec (<http://alitec.sg>)
```

Comment:
You should only put yourself on CONTRIBUTORS.md. Authorship is for major changes.

Reply:
@rousseldenis  This all requested changes are updated.